climada/hazard/centroids/centr.py

```diff
@@ -331,11 +331,16 @@ def from_pnt_bounds(cls, points_bounds, res, crs=DEF_CRS):
             }
         )
 
-    def append(self, centr):
-        """Append Centroids
+    def append(self, *centr):
+        """Append Centroids to the current centroid object for concatenation.
+
+        This method check that all centroids use the same CRS, append the list of centroids to
+        the initial Centroid object and eventually concatenate them to create a single centroid
+        object with the union of all centroids.
 
         Note that the result might contain duplicate points if the object to append has an overlap
-        with the current object.
+        with the current object. Duplicates points will be removed in `union`
+        by calling `remove_duplicate_points`.
```

Comment:
```suggestion
        with the current object. Remove duplicates by either using :py:meth:`union`
        or calling :py:meth:`remove_duplicate_points` after appending.
```
This should render correctly in the docs, but one needs to verify.